lib/lotus/commands/db.rb

```diff
@@ -0,0 +1,59 @@
+require 'lotus/router'
+require 'lotus/model'
+
+module Lotus
+  module Commands
+    class DB
+      attr_reader :environment, :options, :name
```

Comment:
What do you think about making these readers private? I don't see any reason behind exposing it to the outside world at this stage.